config/pipeline.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+module.exports = {
+    AttributeDefinitions: [ /* required */
```

Comment:
Or at least the columns that we would index. There is planned work to be able to extend the `data-schema` so that we don't need to hard-code it here. This is included just so we can get an initial attempt working.